src/commands/transaction/get.js

```diff
@@ -85,6 +162,17 @@ GetCommand.args = [
 GetCommand.flags = {
 	...BaseCommand.flags,
 	state: flagParser.string(stateFlag),
+	senderId: flagParser.string(senderIdFlag),
+	limit: flagParser.string({
+		description:
+			'Limits the returned transactions array by specified integer amount. Maximum is 100.',
+		default: '20',
```

Comment:
from the documentation, it seems like default is `10`
https://lisk.io/documentation/lisk-core/user-guide/api/1-0#/Transactions/getTransactions

Reply:
@shuse2 I just checked the API and actually, it doesn't support sort by height. `Available values : amount:asc, amount:desc, fee:asc, fee:desc, type:asc, type:desc, timestamp:asc, timestamp:desc. Default value : amount:asc`
We can also keep it default that will sort it by amount or we keep it by height?